### ISSUE TITLE
Colour contrast accessibility fail on the jobs header

### ISSUE THREAD

**Issue by idavidmcdonald** [concourse/concourse#7752]
## Summary

A recent change of colour and styling on the jobs page makes reading the text incredibly hard due to the colour contrast between the text and the background.

<img width="663" alt="show-available-pipeline-variables" src="https://user-images.githubusercontent.com/7228605/139670488-fcbf38aa-a498-4f4f-9511-b9459f0e142f.png">

Not only do I personally find it extremely difficult to read, it is also a fail under the WCAG AA accessibility criteria.

## Context

I believe the colours/CSS styles on the running job page have recently changed. I didn't delve too far in to the internals of the code as I'm not an expert but the likely contender appears to be  https://github.com/concourse/concourse/commit/f4b305cbc5766dfbb0272aefd6d164e2c43d6757#diff-b70d655dad31a4f8dc3256614b23b8ff5ec88b8cac8cbf62b950ae75459d2e10 by @xtremerui, @EstebanFS and @taylorsilva

By comparing Concourse v 7.2.0 with v7.6.0 I can visually see a difference, for example

### v7.2.0
<img width="286" alt="build-admin-pr #2500" src="https://user-images.githubusercontent.com/7228605/139670505-9e3519b5-a368-46b2-bee7-caeae7794467.png">

### v7.6.0
<img width="663" alt="show-available-pipeline-variables" src="https://user-images.githubusercontent.com/7228605/139670488-fcbf38aa-a498-4f4f-9511-b9459f0e142f.png">

It appears that the yellow in v7.2.0 is #F1C40F and now in v7.6.0 is #FAD43B

In both, the colour of the text remains as #E2E2E2.

Both of these contrasts fail the accessibility criteria for WCAG AA and AAA - https://webaim.org/resources/contrastchecker/ so the accessibility fail is not a new bug, however the recent change appears to have made it much worse.

This issue seems related to, but not dealt with as part of, https://github.com/concourse/concourse/issues/5974

## Triaging info

* Concourse version: 7.6.0
* Browser (if applicable): Tested in Chrome only but believe this is not a browser specific issue
* Did this used to work? No but appears to have now gotten worse

**Comment by xtremerui** [comment]
I think making the job name black as the timestamp `started 5s ago` would help the readability.

**Comment by idavidmcdonald** [comment]
Making it black like the job name would also pass the accessibility criteria
![image](https://user-images.githubusercontent.com/7228605/139866406-0459fd3d-e06f-4d84-a1f5-cb3ad505ff62.png)
